src/Lykke.Service.SmsSender/Sagas/Commands/ProcessSmsCommand.cs

```diff
@@ -0,0 +1,13 @@
+﻿using ProtoBuf;
+
+namespace Lykke.Service.SmsSender.Sagas.Commands
+{
+    [ProtoContract]
+    public class ProcessSmsCommand
```

Comment:
we can add requariment here - send ClientId?
will be good in log write info about clientId, becouse if we have problem with client, we can found this event by ClientId

Reply:
currently, we don't have clientId passed to the service. Need to change the other services for this. Can be implemented in a separate task